lib/tasks/user.rake

```diff
@@ -0,0 +1,21 @@
+namespace :user do
+  desc "update role field production data"
+  task update_role: :environment do
+    User.all.each do |user|
+      enum_role = check_role(user.role)
+      user.update(role: enum_role)
+    end
+
+    puts 'User role updated'
+  end
+
+private
+  def check_role(role)
+    case role
+    when 'admin'
+      0
```

Comment:
I think we'll need to reverse this based on the enum pecking order in #670, but I see what you mean.